lib/openstudio-standards/standards/necb/ECMS/nv.rb

```diff
@@ -0,0 +1,187 @@
+                zone_hvac_equipment_list.addEquipment(zn_vent_design_flow_rate_1)
+
+                ##### Add another "ZoneVentilation:DesignFlowRate" object for NV to set OA per floor area.
+                zn_vent_design_flow_rate_2 = OpenStudio::Model::ZoneVentilationDesignFlowRate.new(model)
+                zn_vent_design_flow_rate_2.setDesignFlowRateCalculationMethod('Flow/Area')
+                zn_vent_design_flow_rate_2.setFlowRateperZoneFloorArea(oa_per_floor_area_normalized_by_number_of_windows)
+                zn_vent_design_flow_rate_2.setVentilationType('Natural')
+                zn_vent_design_flow_rate_2.setMinimumIndoorTemperatureSchedule(min_Tin_schedule)
+                zn_vent_design_flow_rate_2.setMaximumIndoorTemperatureSchedule(max_Tin_schedule)
+                zn_vent_design_flow_rate_2.setMinimumOutdoorTemperature(nv_Tout_min)
+                zn_vent_design_flow_rate_2.setMaximumOutdoorTemperatureSchedule(max_Tin_schedule)
+                zn_vent_design_flow_rate_2.setDeltaTemperature(nv_Delta_Tin_Tout)
+                zone_hvac_equipment_list.addEquipment(zn_vent_design_flow_rate_2)
+
+                ##### Add the "ZoneVentilation:WindandStackOpenArea" for NV.
+                # Note: it has been assumed that 'Opening Effectiveness' and 'Discharge Coefficient for Opening' are autocalculated (which are the default assumptions).
+                zn_vent_wind_and_stack = OpenStudio::Model::ZoneVentilationWindandStackOpenArea.new(model)
+                zn_vent_wind_and_stack.setOpeningArea(window_area * nv_opening_fraction)
+                zn_vent_wind_and_stack.setOpeningAreaFractionSchedule(operable_window_schedule)
+                # (Ref: E+ I/O) The Effective Angle value "is used to calculate the angle between the wind direction and the opening outward normal to determine the opening effectiveness values when the input field Opening Effectiveness = Autocalculate."
+                # (Ref: E+ I/O) "Effective Angle is the angle in degrees counting from the North clockwise to the opening outward normal."
+                zn_vent_wind_and_stack.setEffectiveAngle(window_azimuth_deg)
+                zn_vent_wind_and_stack.setMinimumIndoorTemperatureSchedule(min_Tin_schedule)
+                zn_vent_wind_and_stack.setMaximumIndoorTemperatureSchedule(max_Tin_schedule)
+                zn_vent_wind_and_stack.setMinimumOutdoorTemperature(nv_Tout_min)
+                zn_vent_wind_and_stack.setMaximumOutdoorTemperatureSchedule(max_Tin_schedule)
+                zn_vent_wind_and_stack.setDeltaTemperature(nv_Delta_Tin_Tout)
+                zone_hvac_equipment_list.addEquipment(zn_vent_wind_and_stack)
+
+              # elsif nv_comfort_model == 'Adaptive_Model'  #TODO: QUESTION: do we want to include adaptive thermal comfort model (ASHRAE 55)?
```

Comment:
Not sure.. Does this control when users open the window? Would it interfere with the deltaT control?

Reply:
Removed the nv_comfort_model argument and the relevant if-clause.